sirius/web/templates/admin.html

```diff
@@ -1,12 +1,9 @@
-{% extends "base.html" %}
-
-{% block title %}Sirius - Admin{% endblock %}
-
-{% block page_content %}
+{% extends "base.html" %} {% block title %}Sirius - Admin{% endblock %} {% block
+page_content %}
```

Comment:
This formatting looks a little broken to me.... does Jinja allow a line break in a directive? Also, it would be much preferred to put the `extends` directive on a new line, if at all possible

Reply:
Jinja _allows_ the line breaks, but [Prettier](https://prettier.io/) is only parsing/formatting it as HTML. There's a(n open) ticket for Jinja support (https://github.com/prettier/prettier/issues/5754), and a link to a Nunchuks based plugin (which is incomplete).

From what I can tell, any of the Python specific formatters (black, autopep8, yapf) won't touch the HTML part of Jinja, and Prettier won't do much to consider the Python parts (just format them as blobs). Given that there's more HTML than Python here, I'm more inclined to favour getting the HTML readable.

We _could_ manually format the Jinja specific lines and throw in `<!-- prettier-ignore -->`, but that might just be more trouble than it's worth?

I'll take a look at the Nunchucks plugin - it might have enough of the simple functionality to at least format the Jinja blocks sequentially, instead of a single line.

Reply:
fwiw the Nunchuks plugin has some legs, but also has some bugs. It'll autoformat that block as:

<img width="237" alt="image" src="https://user-images.githubusercontent.com/33126/66791751-589c4c80-eeaa-11e9-8a9f-42676a6903ec.png">

/me adds it to the list